addons/resource.language.en_gb/resources/strings.po

```diff
@@ -12509,7 +12509,11 @@ msgctxt "#24034"
 msgid "Check for updates"
 msgstr ""
 
-#empty string with id 24035
+#: Name of an Add-on category
+#: xbmc/addons/Addon.cpp
+msgctxt "#24035"
+msgid "Image collections"
```

Comment:
I'd be tempted to go with "image pack" as well though "image collection" works as well. Don't let that hold up the PR :)